src/ext/ddtrace.c

```diff
@@ -224,10 +283,80 @@ static PHP_FUNCTION(dd_trace) {
         }
     }
 
-    zend_bool rv = ddtrace_trace(class_name, function, callable TSRMLS_CC);
+    zend_bool rv = ddtrace_trace(class_name, function, callable, 0 TSRMLS_CC);
+    RETURN_BOOL(rv);
+}
+
+static PHP_FUNCTION(dd_trace_method) {
```

Comment:
I was thinking recently that we don't use Namespaces. Would it make users more happy if we did not "pollute" global namespace ? At very least would it make sense to explicitly define functions in root namespace ?
Its a good opportunity to consider it since we're introduction quite a few new functions .
🤔 `DDTrace/trace_method` vs `dd_trace_method`

Reply:
Also what about php 5.3 :duck:`s

Reply:
I'm totally down for using namespaces. :) I would prefer we update all the API's at once if we're going to do it vs having some functions under the `DDTrace/` namespace and others with a `dd_trace_` prefix. But I'm totally +1 on the namespace idea for a future PR. :)

Reply:
Just keep in mind, that we do need to have some backwards compatibilty, since those are publicly accessible functions, and we **can not break the contract** from version to version.

Simply introducing a namespace at this point you would avoid the need to also transition these new functions without declaring them twice during transition period. 
But ok, it was only an optional suggestion.